Harvest.Web/ClientApp/src/Tickets/TicketTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { useMemo } from "react";
+import { Cell, Column, TableState } from "react-table";
+import { ReactTable } from "../Shared/ReactTable";
+import { ReactTableUtil } from "../Shared/TableUtil";
+import { Ticket } from "../types";
+import { Link } from "react-router-dom";
+
+interface Props {
+  tickets: Ticket[];
+}
+
+export const TicketTable = (props: Props) => {
+  const ticketData = useMemo(() => props.tickets, [props.tickets]);
+  const columns: Column<Ticket>[] = useMemo(
+    () => [
+      {
+        Cell: (data: Cell<Ticket>) => (
+          <div>
+            <Link
+              to={`/ticket/details??projectId=${data.row.original.projectId}&id=${data.row.original.id}`}
+            >
+              #{data.row.original.id}
+            </Link>
+          </div>
+        ),
+        Header: " ",
+        maxWidth: 150,
+      },
+      {
+        Header: "Status",
+        accessor: (row) => row.status,
+      },
+      {
+        Header: "Created",
+        accessor: (row) =>
+          row.createdOn ? new Date(row.createdOn).toLocaleDateString() : "",
```

Comment:
is created really optional?

Reply:
No, but i wasn't sure when I added it to types if it wasn't optional if it would cause problems with the create ticket where this gets set in the controller.